point_viewer_proto_rust/src/proto.proto

```diff
@@ -75,23 +75,47 @@ message NodeId {
   int64 deprecated_index = 2;
 }
 
-message Node {
-  enum PositionEncoding {
+enum PositionEncoding {
     INVALID = 0;
     Uint8 = 1;
     Uint16 = 2;
     Float32 = 3;
     Float64 = 4;
-  }
+}
 
+message Node {
```

Comment:
Nit: If we're making breaking changes anyway, could this become OctreeNode?

Reply:
done